pkg/pipeline/extract/aggregate/aggregate.go

```diff
@@ -117,6 +129,7 @@ func (aggregate Aggregate) UpdateByEntry(entry config.GenericMap, normalizedValu
 
 	if operation == OperationCount {
 		groupState.value = float64(groupState.count + 1)
+		groupState.recentOpValue = float64(groupState.recentCount + 1)
 		groupState.RecentRawValues = append(groupState.RecentRawValues, 1)
```

Comment:
s/RecentRawValues/recentRawValues .... no reason to expose + make sure we are consistnet